docs/app/pages/components/components-sections/media-player/media-player/media-player.component.html

```diff
@@ -1,13 +1,12 @@
 <ux-media-player #player *ngIf="type === 'video'" class="media-player" type="video" [source]="videoSource" [quietMode]="mode === 'quiet'">
     <track label="English" lang="en" default [src]="subtitles">
 </ux-media-player>
-<ux-media-player *ngIf="type === 'audio'" class="media-player" type="audio" [source]="audioSource"></ux-media-player>
+<ux-media-player *ngIf="type === 'audio'" class="media-player" type="audio"  [source]="audioSource" [filename]="filename"></ux-media-player>
```

Comment:
I think we decided not to include this in the documentation example, in which case we can remove the filename attribute from here and the variable from the component typescript file